layouts/shortcodes/redborder.html

```diff
@@ -0,0 +1 @@
+<p style="padding: 5px; background-color: red;">{{.Inner}}</p>
```

Comment:
Seriously we need a whole file for such a specific operation?

Reply:
Yes, that's how hugo operates. 😞

Reply:
ohh....so good hear something like that from the only Hugo advocate currently lol :rofl:

Reply:
But hugo has built in shortcodes as well. 😃